app/src/app/components/header/AccountHeaderDropdown.tsx

```diff
@@ -12,31 +11,31 @@ import {
   DropdownMenuSeparator,
   DropdownMenuTrigger
 } from "../Base/DropdownMenu";
+import { useUser } from "../providers/UserProvider";
 
-export default function AccountHeaderDropdown() {
+export function AccountHeaderDropdown() {
   const navigate = useNavigate();
-  const dispatch = useAppDispatch();
+  const { removeUser, user } = useUser();
 
   const handleLogout = () => {
-    dispatch(logout());
+    removeUser();
     navigate("/login");
   };
 
   return (
-    <DropdownMenu data-testid="test1">
+    <DropdownMenu>
       <DropdownMenuTrigger asChild>
         <Button variant="ghost" className="relative h-10 w-10 rounded full">
           <Avatar className="h-8 w-8">
-            <AvatarFallback>LA</AvatarFallback>
+            <AvatarFallback>{getInitials(user?.displayName ?? "X X")}</AvatarFallback>
```

Comment:
this is probably causing failures
```suggestion
            <AvatarFallback>{getInitials(user?.displayName || "X X")}</AvatarFallback>
```

Reply:
although thinking about it again, I think the `getInitials` function should be the one setting the default maybe rather than throwing an error?